controllers/workflow_controller.go

```diff
@@ -244,6 +266,28 @@ func (r *WorkflowReconciler) createComputes(ctx context.Context, wf *dwsv1alpha2
 	return computes, nil
 }
 
+// Next reports the next state after state s
```

Comment:
state => status?   The "next" feels odd.
I'm confused about what the numbers mean in this function.

Reply:
Seems like this comment is meant for something else?